include/Makefile

```diff
@@ -458,7 +458,7 @@ symlinks: .PHONY .META
 	cd ${SRCTOP}/sys/rpc;
 .if ${MK_CDDL} != "no"
 	${INSTALL_SYMLINK} ${TAG_ARGS:D${TAG_ARGS},dev} \
-	    ../../..${ZFSTOP:C!^${SRCTOP}!!}/include/sys/nvpair.h \
```

Comment:
This should be fine as it was?

Reply:
Yeah, seems to work here, I think I'd ran into issues with an attempt to arrange a patch to upstream and didn't carried that over.